members/sophiePark.md

```diff
@@ -0,0 +1,15 @@
+# Sophie Se Hyun PARK
+
+## Education
+
+* National University of Singapore, Singapore (B.Comp in Computer Science)
+
+## Work Experience
+
+* [3 months: 2017 May - July] Graphic Designer and Web Developer Intern, Miao Technology Pte Ltd, Singapore
+* [Freelance] Korean-English Translator
+
+## Accomplishments
```

Comment:
Might want to include skills and capabilities too.
I.e Programming languages proficiency, etc.

Reply:
Reflected in commit!